device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/edge/DirectMethodResponse.java

```diff
@@ -50,21 +50,21 @@ public JsonElement getPayloadAsJsonElement()
     }
 
     /**
-     * Return the DirectMethodResponse payload in String type
-     * @return the DirectMethodResponse payload in String type
+     * Use this if you wish to deserialize to a specific type using a deserialization library of your choice
+     * Return the DirectMethodResponse payload in json string
```

Comment:
I'd swap the order of these statements. The first should be the brief summary and the second should be the extra info.